src/docs/schema.dox

```diff
@@ -362,4 +362,16 @@ Here is another example program, @ex_ref{ex_call_center.c}.
 @snippet ex_call_center.c call-center decl
 @snippet ex_call_center.c call-center work
 
+@section logged_non_logged_tables Logged and Non-logged tables
+
+- WiredTiger writes records to the log for each transaction when logging is enabled
+  using the log=(enabled) configuration to wiredtiger_open.
```

Comment:
np: To help distinguish them as code/configs, might be worth formatting `log=(enabled) ` and `wiredtiger_open` with `\c log=(enabled) ` and `\c wiredtiger_open` (respectively).

Reply:
Thanks, @alisonfel for flagging, updated the comments.